src/main/java/utility/ColorSupplier.java

```diff
@@ -0,0 +1,14 @@
+package utility;
+
+import java.util.Random;
+import model.Color;
+
+public class ColorSupplier {
+    private static final Random random = new Random();
+    private static final Color[] colors = Color.values();
+
+    public String getRandomColor() {
```

Comment:
Consider removing the `final` keyword from the local variable `index` as it's not necessary in this context. It's good practice to use `final` for class fields and method parameters that should not change, but for local variables, it's often superfluous unless you're dealing with closures (like anonymous classes or lambda expressions).